gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/resources/GeneratedResClassSpec.kt

```diff
@@ -283,11 +286,17 @@ private fun getChunkFileSpec(
                 .endControlFlow()
                 .build()
 
-            val accessor = PropertySpec.builder(resName, type.getClassName(), resModifier)
+            val accessorBuilder = PropertySpec.builder(resName, type.getClassName(), resModifier)
                 .receiver(ClassName(packageName, resClassName, type.accessorName))
                 .delegate(initializer)
-                .build()
-            chunkFile.addProperty(accessor)
+            if (System.getProperty("compose.resources.generate.ResourceContentHash.annotation") == "true") {
```

Comment:
I would be interested to learn more about it, having to be enabled? Especially if the content hash is just 32-bit, the memory concern is a really lightweight argument? Am I missing something?

Reply:
It is just unnecessary overhead that we would like to avoid for final distributions. We can strip the annotations in our tooling but there are chances that in a custom build they would be present. So it seems it is better just not generate the annotations but for Hot Reload.